src/mpid/ch4/include/coll_algo_params.h.in

```diff
@@ -12,135 +12,135 @@
 
 typedef union MPIDI_Barrier_algo_params {
     /* *INDENT-OFF* */
-    MPIDI_CH4_BARRIER_PARAMS_DECL;
+    MPIDI_CH4I_BARRIER_PARAMS_DECL;
```

Comment:
At some point we were in agreement that `MPIDI_CH4I` is redundant given that only one device is built at a time. Therefore all of these instances can be renamed `MPIDI_CH4I` -> `MPIDI`. Can you make that change instead?